UpdateScripts/V301_drop_functs_replaced_by_procedures.sql

```diff
@@ -15,6 +15,10 @@ DROP FUNCTION IF EXISTS mv$grantSelectPrivileges;
 DROP FUNCTION IF EXISTS mv$insertPgMviewLogs;
 DROP FUNCTION IF EXISTS mv$removeRow$FromSourceTable;
 DROP FUNCTION IF EXISTS mv$truncateMaterializedView;
+DROP PROCEDURE IF EXISTS mv$createindexestemptable;
+DROP PROCEDURE IF EXISTS mv$dropmvindexes;
```

Comment:
This script is for an previous patch change for dropping any functions that no longer required. Anything you need to drop should be in the next version V501_ for example then it goes from a patching point of view in logical order. Sounds like it's just the full one you'd want to drop as none of the new one's will exist